README.rst

```diff
@@ -205,13 +205,26 @@ You must have `Docker installed <https://docs.docker.com/engine/installation/>`_
 
         docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p443:443 -i quipucords:1.0.0
 
+    D. For debugging purposes you may want to run the Docker image with the /app directory mapped to your local clone of quipucords and the logs mapped to a temporary directory::
+
+        docker run -d --name quipucords -e "QPC_DBMS=sqlite" -p443:443 -v /path/to/local/quipucords/:/app -v /tmp:/var/log -i quipucords:1.0.0
```

Comment:
:heart:

Reply:
❤️

Reply:
This maps the application code not logs.  Let's change the labels from `1.0.0` to `master`.  @elyezer started us down this path with Jenkins.  Really 1.0.0 doesn't make a whole lot of sense anymore.  

I also don't know if mapping the logs is that important since you have to docker exec to restart gunicorn anyway.  We may want to just focus on mapping the app source code.  The developer will be in the docker container already so tailing the logs doesn't seem difficult.

Reply:
:+1: for changing from `1.0.0` to `master`

Reply:
This does map the logs @kevan with ```-v /tmp:/var/log```. I agree with @elyezer that the this can be useful. Even though it is not difficult to docker exec into the container to look at the logs, I found it easier to just have it tailing the logs on my host filesystem so that when I was inside the container, it was solely to restart the server.